packages/magento-product/components/AddProductsToCart/AddProductsToCartSnackbarMessage.tsx

```diff
@@ -0,0 +1,84 @@
+import { ApolloError } from '@graphcommerce/graphql'
+import { CartUserInputErrorType, CheckoutUserInputErrorCodes } from '@graphcommerce/graphql-mesh'
+import { ApolloCartErrorSnackbar } from '@graphcommerce/magento-cart'
+import {
+  Button,
+  ErrorSnackbar,
+  ErrorSnackbarProps,
+  IconSvg,
+  ListFormat,
+  MessageSnackbar,
+  MessageSnackbarProps,
+  iconChevronRight,
+} from '@graphcommerce/next-ui'
+import { Plural, Trans } from '@lingui/macro'
+
+export type AddProductsToCartSnackbarMessageProps = {
+  errorSnackbar?: Omit<ErrorSnackbarProps, 'open'>
+  successSnackbar?: Omit<MessageSnackbarProps, 'open' | 'action'>
+  error?: ApolloError
+  userErrors?: { code: CartUserInputErrorType | CheckoutUserInputErrorCodes; message: string }[]
```

Comment:
Can we reduce this to only be `Pick<CartUserInputError, 'message'>`? Else we polute the type with the Reorder stuff here.

Reply:
Fixed